Automattic/simplenote-macos
# Editor: Default Line Length

### Fix
By default, now, Simplenote's Line Length will be set to **enabled**.

cc @SylvesterWilmott (Sir!! just wanted to confirm if this is a go!)
cc @eshurakov 

Thank you both!

Closes #762

### Test: Legacy Narrow
1. Install `develop`
2. Toggle `View > Line Length > Narrow`
3. Install this branch on top

- [ ] Verify the Editor is in Narrow Mode


### Test: Nü Default
1. Run this snippet in **zsh**:
  ```defaults delete com.automattic.SimplenoteMac.beta kEditorWidthPreferencesKey```
2. Install this branch and launch Simplenote

- [ ] Verify `View > Line Length > Full` is selected
- [ ] Verify the Editor's Line Length, effectively, edge to edge

### Release
`RELEASE-NOTES.txt` was updated in 80a0cea with:

> Full Width Line Length is now enabeld by default #762
